### ISSUE TITLE
low accuracy when fine tuning for the MRPC task with large model

### ISSUE THREAD

**Issue by syu0000** [huggingface/pytorch-transformers#722]
I noticed that on the website you said:"Here is an example using distributed training on 8 V100 GPUs and Bert Whole Word Masking model to reach a F1 > 92 on MRPC." However, when I fine tuned the model with max_sequence_length=128 and batch_size=12 on a single 11G GPU, it gives the accuracy of 0.68.
acc = 0.6838235294117647
acc_and_f1 = 0.7480253018237863
eval_loss = 0.6240295206799227
f1 = 0.8122270742358079
global_step = 918
loss = None
I wonder what have let to that.
the command I used "python  run_classifier.py   --bert_model bert-large-uncased-whole-word-masking    --task_name MRPC --do_train   --do_eval   --do_lower_case   --data_dir E:\Users\..... --max_seq_length 64   --train_batch_size 12   --learning_rate 2e-5   --num_train_epochs 3.0  --output_dir E:\Users\...."

**Comment by thomwolf** [comment]
The batch size will be 8 times smaller with only one GPU, increase it by a factor of 8 using gradient accumulation, e.g. `--train_batch_size 96 --gradient_accumulation_steps 8`

**Comment by syu0000** [comment]
Thank you for your help. However, when I use this command: 
python  run_classifier.py   --bert_model bert-large-uncased-whole-word-masking  --task_name MRPC –-do_train --do_eval --do_lower_case --data_dir E:\Users\...\MRPC --max_seq_length 128 --train_batch_size 96 --gradient_accumulation_steps 8 --learning_rate 2e-5 --num_train_epochs 3.0  --output_dir E:\Users\...\MRPC_result
it still gives me a too low acc result.
btw I am not using the latest version of package, could that be the cause?
